L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -48,13 +48,6 @@ HGCalHistoSeedingImpl::HGCalHistoSeedingImpl(const edm::ParameterSet& conf)
       << "\nMulticluster MIPT threshold for histo threshold algorithm: " << histoThreshold_
       << "\nMulticluster type of multiclustering algortihm: " << seedingAlgoType_;
 
-  if (seedingAlgoType_.find("Histo") != std::string::npos && seedingSpace_ == RPhi &&
```

Comment:
There should be checks of binning and sumHisto compatibility in the constructor.